docs/examples/how-tos.md

```diff
@@ -0,0 +1,396 @@
+} );
+```
+
+### How to get the editor instance object from the DOM element?
+
+If you have a reference to the editor editable's DOM element (the one with the `.ck-editor__editable` class and the `contenteditable` attribute), you can access the editor instance this editable element belongs to using the `ckeditorInstance` property:
+
+```html
+<!-- The editable element in the editor's DOM structure. -->
+<div class="... ck-editor__editable ..." contenteditable="true">
+	<!-- Editable content. -->
+</div>
+```
+
+```js
+// A reference to the editor editable element in the DOM.
+const domEditableElement = document.querySelector( '.ck-editor__editable' );
+
+// Get the editor instance from the editable element.
+const editorInstance = domEditableElement.ckeditorInstance;
+
+// Use the editor instance API.
+editorInstance.setData( '<p>Hello world!<p>' );
+```
+
+### How to list all instances of the editor?
+
+By default, CKEditor 5 has no global registry of editor instances. But if necessary, such a feature can be easily implemented, as explained in this [Stack Overflow answer](https://stackoverflow.com/a/48682501/1485219).
+
+## Editor's API
+
+### How to insert some content into the editor?
+
+Because CKEditor 5 uses a custom {@link framework/architecture/editing-engine data model}, whenever you want to insert anything, you should modify the model first, which is then converted back to the view where the users input their content (called "editable"). In CKEditor 5, HTML is just one of many possible output formats. You can learn more about the ways of changing the model in the {@link framework/architecture/editing-engine#changing-the-model dedicated guide}.
+
+For example, to insert a new link at the current position, use the following snippet:
+
+```js
+editor.model.change( writer => {
+	const insertPosition = editor.model.document.selection.getFirstPosition();
+
+	writer.insertText(
+		'CKEditor 5 rocks!',
+		{ linkHref: 'https://ckeditor.com/' },
+		insertPosition
+	);
+} );
+```
+
+And to insert some plain text, you can use a slightly shorter one:
+
+```js
+editor.model.change( writer => {
+	writer.insertText(
+		'Plain text',
+		editor.model.document.selection.getFirstPosition()
+	);
+} );
+```
+
+You may have noticed that a link is represented as a text with an attribute in the editor model. See the API of the {@link module:engine/model/writer~Writer model writer} to learn about other useful methods that can help you modify the editor model.
+
+To insert some longer HTML code, you can parse it to the {@link module:engine/model/documentfragment~DocumentFragment model fragment} first and then {@link module:engine/model/model~Model#insertContent insert} it into the editor model:
+
+```js
+const content =
+	'<p>A paragraph with <a href="https://ckeditor.com">some link</a>.</p>';
+const viewFragment = editor.data.processor.toView( content );
+const modelFragment = editor.data.toModel( viewFragment );
+
+editor.model.insertContent( modelFragment );
```

Comment:
Here we are mentioning `insertContent()` and this part is ok. But what I miss here is the explanation that only elements that are handled by the editor (that have a conversion to the model) will be handled. If some element or attribute does not have declared converters (whether by the dedicated feature or GHS) then those won't get inserted. Maybe it's a good place to explain the "feature" approach to the content, not to the HTML that is building those features.

Reply:
We can add an inline code comment, but intentionally we don't plan to insert longer explanations here, and treat them strictly as task-oriented guides.

Reply:
We can link here: http://localhost:8080/ckeditor5/latest/features/index.html#why-does-the-editor-filter-out-my-content-styles-classes-elements